server/Player.js

```diff
@@ -164,15 +186,14 @@ class Player {
     this.toRespawn = false;
     this.hp = 100;
 
-    const respawnXY = getXYFromTile(respawnTile.x, respawnTile.y);
-    this.positionTile = { tileX: respawnTile.x, tileY: respawnTile.y };
+    const respawnXY = getXYFromTile(respawnTile.tileX, respawnTile.tileY);
+    this.positionTile = respawnTile;
     this.x = respawnXY.x;
     this.y = respawnXY.y;
   }
 
   resetSelected() {
-    this.selectedPlayer = null;
-    this.selectedPlayerTile = null;
+    this.selectionToDrop = true;
```

Comment:
Name suggests that there could be multiple selections to drop, rename to `dropSelection`?